config/master/aws-k8s-cni-cn.yaml

```diff
@@ -156,6 +156,8 @@
           "value": "false"
         - "name": "ENABLE_PREFIX_DELEGATION"
           "value": "false"
+        - "name": "ENI_MANAGE_MODE"
```

Comment:
i feel we don't need to add all env to our manifest if it have a default value in code

Reply:
Yeah but generally we add it so that it is clearer for the reader.